pages/operations/managing-applications/update-application.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Update an Application
+description: How to change an application's version.
+---
+
+Updates to Plural applications are applied **automatically** based on a per-application setting. This setting supports telling Plural to only upgrade for Stable or Warm releases, as well as disabling automatic upgrades altogether.
```

Comment:
Should we explain what Stable/Warm/Latest is for users, or do we think they'll know?

Reply:
Good point, they probably won't know, since I only really understand latest and warm as of writing this comment, I'll add a note in.